src/inspect_ai/model/_call_tools.py

```diff
@@ -545,6 +545,8 @@ def report_parse_error(ex: Exception) -> None:
     return ToolCall(
         id=id,
         function=function,
+        # TODO: ???
+        native_name=None,
```

Comment:
I don't understand the context here, and therefore don't really know if I need to have a `native_name`